benchmark-framework/src/main/java/io/openmessaging/benchmark/WorkloadGenerator.java

```diff
@@ -357,6 +358,8 @@ private void buildAndDrainBacklog(List<String> topics) throws IOException {
             }
         }
 
+        log.info("--- Completed backlog build in {} ms ---", timer.elapsedMillis());
```

Comment:
nit: since this is generally a longer time, we could add an `elapsedSeconds()` to the timer class.

Reply:
Good idea - PTAL